benchmarks/apps/conquest/conquest_runonly.py

```diff
@@ -0,0 +1,116 @@
+# Import modules from reframe and excalibur-tests
+import reframe as rfm
+import reframe.utility.sanity as sn
+from benchmarks.modules.utils import SpackTest
+
+class ConquestBaseBenchmark(rfm.RunOnlyRegressionTest):
+
+    # Run configuration
+    ## Mandatory ReFrame setup
+    valid_systems = ['-gpu']
+    valid_prog_environs = ['default']
+
+    ## Executable
+    executable = 'Conquest'
+    executable_opts = ['']
+
+    ## Scheduler options
+    time_limit = '30m'
+
+    input_dir = variable(str, value='foo')
+
+    # Build configuration
+    # spack_spec = 'conquest@develop +openmp'
+
+    @run_after('setup')
+    def setup_variables(self):
+        self.env_vars['OMP_NUM_THREADS'] = f'{self.num_cpus_per_task}'
+
+        if self.current_partition.scheduler.registered_name == 'sge':
```

Comment:
Why do you need to do this here?

Reply:
It's normally done in the `SpackTest` base class but this is a `Rfm.RunOnlyRegressionTest`

Reply:
Ah, I missed you were deriving from `Rfm.RunOnlyRegressionTest`.